ccnmtldjango/template/+package+/main/tests/test_views.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
 
     def test_page(self):
         r = self.c.get("/pages/section-1/")
-        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.status_code, 403)
```

Comment:
Because of this change in pagetree: https://github.com/ccnmtl/django-pagetree/pull/17

Maybe the check I added there needs to go below the check for `if not self.get_gated()` so it doesn't affect pagetree views that have `gated` set to `False`?